### PR TITLE
Option for generating sample TLS certs for Minio GW

### PR DESCRIPTION
**Pull request by mniewrzal**
New option for captplanet/uplink setup will use uplink identity key and cert to provide sample TLS certs for Minio GW.

Jira: V3-392

### CODE REVIEW

File: pkg/peertls/templates.go [hunk 1]
```diff
@@ -20,6 +21,7 @@ func CATemplate() (*x509.Certificate, error) {
 		ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
 		BasicConstraintsValid: true,
 		IsCA:                  true,
+		Subject:               pkix.Name{CommonName: "Storj"},
```
Comment (egonelbre) [inline pkg/peertls/templates.go]: Subject CN is pretty much deprecated, I think. https://security.stackexchange.com/a/69160

Comment (mniewrzal) [reply pkg/peertls/templates.go]: Ok, it looks that I need one from three (C,O,CN) to be able to use this cert with minio and Filezilla PRO. I will use 'Organization'. Thanks

File: cmd/captplanet/setup.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ type Config struct {
 	APIKey              string `default:"abc123" help:"the api key to use for the satellite"`
 	EncKey              string `default:"highlydistributedridiculouslyresilient" help:"your root encryption key"`
 	Overwrite           bool   `help:"whether to overwrite pre-existing configuration files" default:"false"`
-	MinioTLS            bool   `default:"false" help:"generate sample TLS certs for Minio GW"`
+	GenerateMinioCerts  bool   `default:"false" help:"generate sample TLS certs for Minio GW"`
```
Comment (egonelbre) [inline cmd/captplanet/setup.go]: Ah, wait this is also for the Config, not just the setup? Sorry, can you revert this. I just realized the previous one was probably better.

Comment (mniewrzal) [reply cmd/captplanet/setup.go]: What do you mean for Config? It will be generating certs during setup only so you version fits quite well.

Comment (egonelbre) [reply cmd/captplanet/setup.go]: Oh, well. For some reason it seemed wrong. Whichever you think is better.

**Pull request merged**